### ISSUE TITLE
Changes needed for further block support

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/highland-edu-theme#21]
See this page: https://staging.highland.edu/new-block-based-home-page/

1. Add a checkbox on the Full-width page template to "Hide the H1 visually but keep it for screen reader users." That will put make this banner screen reader only and not visible on the page:
<img width="1252" alt="Screen Shot 2023-08-13 at 1 05 59 PM" src="https://github.com/equalizedigital/highland-edu-theme/assets/13153456/7d6e5d8c-7e31-4a16-9ff1-2e3f6b7c94ac">

If that option is set also set .entry-content padding to 0.

2. Fix the strip of white on the lext slide of alignfull. Here are two examples:
<img width="774" alt="Screen Shot 2023-08-13 at 1 25 44 PM" src="https://github.com/equalizedigital/highland-edu-theme/assets/13153456/ff3ab115-dcc0-46b4-9671-6b92f9d4a065">
<img width="1259" alt="Screen Shot 2023-08-13 at 1 26 50 PM" src="https://github.com/equalizedigital/highland-edu-theme/assets/13153456/c0c3a853-7085-43d5-8484-70392c0cb3c1">

3. Adjust theme color palette:
<img width="250" alt="Screen Shot 2023-08-13 at 1 42 57 PM" src="https://github.com/equalizedigital/highland-edu-theme/assets/13153456/f042837b-aa5d-4bf9-a1bb-b3f055fa9397">
   - Remove red
   - Orange should be: #ff6200
   - Blue should be #285598
   - Add Dark blue: #193662
   - Add Brown: #603813
   - Gray should be: #787474
   - Add Mid Gray: #8b8989
   - Add Light Gray: #EBEBEB
   - Add Light Orange: #FFDCC7

4.  Remove the color declaration from strong text:
<img width="354" alt="Screen Shot 2023-08-13 at 2 51 25 PM" src="https://github.com/equalizedigital/highland-edu-theme/assets/13153456/4a3fdd75-fdc4-4230-ba25-db47820d90bc">

5. Button styles:
<img width="2157" alt="Screen Shot 2023-08-13 at 3 11 59 PM" src="https://github.com/equalizedigital/highland-edu-theme/assets/13153456/da8ab1a5-2d5d-4bfb-b5c1-8a4f98436815">
<img width="882" alt="Screen Shot 2023-08-13 at 3 12 25 PM" src="https://github.com/equalizedigital/highland-edu-theme/assets/13153456/1958689a-393d-4d19-8a56-a20d8086153c">
<img width="1296" alt="Screen Shot 2023-08-13 at 3 13 58 PM" src="https://github.com/equalizedigital/highland-edu-theme/assets/13153456/8526e91c-9c44-42d9-a036-2289637306f6">
- All buttons should have a border-radius of 25px by default (without having to set it in the editor)
- Default button background should be Blue (hex above).
- Default button background on hover should be Dark Blue.
- Change the underline on hover to Orange.
- Outline buttons should have this style by default: orange border, white background, dark blue text; on hover make the background orange and the text black (000).
- If an outline button is set to have a dark blue or blue background, set the text-color to white. Hover stays the same as above.

6. Query block image styles not output on the front. These are set to cover:
<img width="1565" alt="Screen Shot 2023-08-13 at 3 23 37 PM" src="https://github.com/equalizedigital/highland-edu-theme/assets/13153456/bf06fdc9-dd9f-4cd0-b783-6543fc00e8d9">
However, it's not working on the front:
<img width="1246" alt="Screen Shot 2023-08-13 at 3 24 43 PM" src="https://github.com/equalizedigital/highland-edu-theme/assets/13153456/6aeea1b9-5d8e-4473-944a-91b6ee683e5c">

7. Cover block also seems not to be covering responsively:
<img width="587" alt="Screen Shot 2023-08-13 at 4 14 18 PM" src="https://github.com/equalizedigital/highland-edu-theme/assets/13153456/c64db037-1e2e-42fd-b09f-8cb266c6d6eb">
